### ISSUE TITLE
Gradle version

### ISSUE THREAD

**Issue by kafushy** [ratrecommends/dice-heroes#5]
Is there a way to actually run this project on pc.
The gradle version 1.2.2 is unsuported everywhere, or is there a way to change it.
Sorry i don't know a lot about programming but i really wanted to examine this game.

**Comment by vlaaad** [comment]
Hmm, do you mean running gradlew does not work? It's been so long since I touched Dice Heroes...

**Comment by kafushy** [comment]
yeah it doesnt work... i know that you dont want to change anything about it, i was just hoping there would be a way to run it, but i dont have nearly enought techinical skills to figure it out by myself
